src/main/java/uk/gov/hmcts/reform/sscs/service/JudicialRefDataService.java

```diff
@@ -54,15 +55,18 @@ private String getJudicialUserFullName(@NonNull String personalCode, IdamTokens
     }
 
     private static String splitInitials(JudicialUser judicialUser) {
-        String initials = judicialUser.getInitials();
-
-        if (isNotBlank(initials)) {
-            StringBuilder result = new StringBuilder();
-            for (int i = 0; i < initials.length(); i++) {
-                result.append(initials.charAt(i));
-                result.append(" ");
+        String fullName = judicialUser.getFullName();
```

Comment:
would it not be simpler to use the initials field or is that not reliable? (minus the surname initial, of course)

Reply:
sorry, could we use the regex for whitespace as well` .split("\s")`
thanks